.github/ISSUE_TEMPLATE/example-bug-report.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
What's the difference between this file and `bug_report`?

Reply:
This file allows creating issues on our Liveblocks examples.
I can delete it if we don't want to split the issue declaration between the liveblocks packages and examples.

Reply:
I've removed this template. 
It can be confusing for our users to have a specific template for liveblocks examples.